.pylintrc

```diff
@@ -0,0 +1,54 @@
+[MASTER]
+# Use multiple processes to speed up PyLint. Specifying 0 will auto-detect the
+# number of processors available to use.
+jobs=0
+
+load-plugins=pylint.extensions.bad_builtin,
+             pylint.extensions.check_elif,
+             pylint.extensions.comparetozero,
+             pylint.extensions.docparams,
+             pylint.extensions.emptystring,
+             pylint.extensions.mccabe,
+             pylint.extensions.overlapping_exceptions,
+             pylint.extensions.redefined_variable_type,
+
+[MESSAGES CONTROL]
+disable=
+    # Docstrings are encouraged but we don't want to enforce that everything
+    # must have a docstring.
+    missing-docstring,
+
+    # We don't always want to have to put a `:return:` in a docstring.
+    missing-return-doc,
+
+    # We don't always want to have to put an `:rtype:` in a docstring.
+    missing-return-type-doc,
+
+    # We don't want to have to document the type of every parameter with a
+    # `:type:` in the docstring.
+    missing-type-doc,
+
+    # We use isort to sort and group our imports, so we don't need PyLint to
+    # check them for us.
+    ungrouped-imports,
+
+    # We use Black to format our code automatically, so we don't need PyLint to
+    # check formatting for us.
+    bad-continuation,
+    line-too-long,
```

Comment:
We have a few more exceptions here than we do in our cookie cutters. I wonder if we want to look at removing one or two in future to get them inline. Certainly better than trying to fix this all now.

It's already plenty better than before.